en/controllers/middleware.rst

```diff
@@ -229,7 +225,9 @@ their own response. We can see both options in our simple middleware::
                     'expire' => '+ 1 year',
                 ]);
             }
-            return $response;
+            
+            // Pass the modified instances to the next Middleware
+            return $next($request, $response);
```

Comment:
Modifying the response should always be done *after* the next layers are invoked, not before.